### PR TITLE
Add additional parameter for CID

### PR DESCRIPTION
**Pull request by ThijsLecomte**
Required items, please complete
   
   Change(s):
   - Enable CID parameter on connector

   Reason for Change(s):
   - Currently one parameter is hard coded which provides issues with customers using this rule

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - OK

   Checked that the validations are passing and have addressed any issues that are present:
   - OK

### CODE REVIEW

**Comment by preetikr** [comment]
@ashishsyal / @v-jayakal - Please review this change + potential solution modification. Thanks.

File: Solutions/LastPass/Data Connectors/LastPassAPIConnector.json [hunk 1]
```diff
@@ -1,123 +1,132 @@
...
+      "name": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',guid(subscription().subscriptionId))]",
+      "apiVersion": "2021-03-01-preview",
+      "type": "Microsoft.OperationalInsights/workspaces/providers/dataConnectors",
+      "kind": "APIPolling",
+      "properties": {
+        "connectorUiConfig": {
+          "title": "LastPass Enterprise - Reporting (test)",
+          "publisher": "The Collective Consulting BV",
+          "descriptionMarkdown": "The [LastPass Enterprise](https://www.lastpass.com/products/enterprise-password-management-and-sso) connector provides the capability to LastPass reporting (audit) logs into Azure Sentinel. The connector provides visibility into logins and activity within LastPass (such as reading and removing passwords).",
+          "graphQueriesTableName": "LastPass_BYOC_CL",
```
Comment (Tichandr) [inline Solutions/LastPass/Data Connectors/LastPassAPIConnector.json:22]: @ThijsLecomte - Can you also change the "graphQueriesTableName" to something like "LastPassNativePoller_CL"?

File: Solutions/LastPass/Data Connectors/LastPassAPIConnector.json [hunk 2]
```diff
@@ -1,123 +1,132 @@
...
+              "query": "{{graphQueriesTableName}}\n | where Action_s == \"Move to Shared Folder\"\n | extend AccountCustomEntity = Username_s, IPCustomEntity = IP_Address_s, URLCustomEntity = Data_s, TimestampCustomEntity = todatetime(Time_s) "
+            }
+          ],
+          "dataTypes": [
+            {
+              "name": "{{graphQueriesTableName}}",
+              "lastDataReceivedQuery": "{{graphQueriesTableName}}\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
             }
+          ],
+          "connectivityCriterias": [
```
Comment (Tichandr) [inline Solutions/LastPass/Data Connectors/LastPassAPIConnector.json:42]: @ThijsLecomte - Please change the property "connectivityCriterias" to "connectivityCriteria". There shouldn't be additional 's' as the end for this property

File: Solutions/LastPass/Data Connectors/LastPassAPIConnector.json [hunk 3]
```diff
@@ -1,123 +1,132 @@
...
+          },
+          "request": {
+            "apiEndpoint": "https://lastpass.com/enterpriseapi.php",
+            "rateLimitQPS": 2,
+            "httpMethod": "Post",
+            "queryTimeFormat": "yyyy-MM-ddTHH:mm:ssZ",
+            "retryCount": 3,
+            "queryWindowInMin": 10,
+            "timeoutInSeconds": 120,
+            "queryParametersTemplate": "{'cid': '{{cidPlaceHolder}}', 'cmd': 'reporting', 'data': { 'from': '{_QueryWindowStartTime}', 'to': '{_QueryWindowEndTime}' }, '{_APIKeyName}': '{_APIKey}'}",
```
Comment (Tichandr) [inline Solutions/LastPass/Data Connectors/LastPassAPIConnector.json:112]: @ThijsLecomte - I hope you have tested the paging and api functionality to retrieve the data to LA workspace.

Comment (ThijsLecomte) [reply Solutions/LastPass/Data Connectors/LastPassAPIConnector.json:113]: Yes, I had this running in production for a while.

**Review by v-sabiraj — APPROVED** [review]

**Pull request merged**